src/wms/src/test/java/org/geoserver/wms/wms_1_1_1/LegendCapabilitiesTest.java

```diff
@@ -128,10 +128,10 @@ public void testCapabilities() throws Exception {
         // two
         // cases
         String expectedGetLegendGraphicRequestURL =
-                "/wms?request=GetLegendGraphic&format=image%2Fjpeg&width=20&height=20&layer=gs%3A"
+                "/wms?request=GetLegendGraphic&format=image%2Fjpeg&width=22&height=22&layer=gs%3A"
```

Comment:
There are constants for these values, best to use them

Reply:
fixed